cg/meta/workflow/mip_dna.py

```diff
@@ -67,3 +70,17 @@ def config_sample(
     def get_gene_panel(self, case_id: str) -> list[str]:
         """Create and return the aggregated gene panel file."""
         return self._get_gene_panel(case_id=case_id, genome_build=GENOME_BUILD_37)
+
+    def get_managed_variants(self) -> list[str]:
+        """Create and return the managed variants."""
+        return self._get_managed_variants(genome_build=GENOME_BUILD_37)
+
+    def write_managed_variants(self, case_id: str, content: list[str]) -> None:
+        """Write the managed variants to case dir."""
+        out_dir = Path(self.root, case_id)
+        out_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
```suggestion
        out_dir = Path(self.root, case_id)
        out_dir.mkdir(parents=True, exist_ok=True)
```
should this directory not already exist? Would an exists check that raises an error be more appropriate? (not sure about the details of the flow though)